cms/static/js/views/course_outline.js

```diff
@@ -216,6 +411,23 @@ function(
             }
         },
 
+        /**
+         * If the new "Actions" menu is enabled, most actions like Configure,
+         * Duplicate, Move, Delete, etc. are moved into this menu. For this
+         * event, we just toggle displaying the menu.
+         * @param {*} event 
+         */
+        showActionsMenu(event) {
```

Comment:
It wasn't apparent in the Unit view, but now I see that we should close a menu when opening another one. Otherwise, we will get this:
![image](https://github.com/openedx/edx-platform/assets/3189670/e8188acb-ef80-4b25-b9ce-a73e2c231e0a)

It's not a blocker - we can do this in a follow-up ticket.

Reply:
Tracking in https://github.com/openedx/modular-learning/issues/92